recipe/meta.yaml

```diff
@@ -35,12 +32,15 @@ requirements:
     - swig
     - libitk-devel
     - libitk <5.2
+    - nlohmann_json [not win]
```

Comment:
here missing a `#` should be 
` -nlohmann_json  # [not win]`

Reply:
yes indeed. I had it ligned up for the next update, but saw that it was going through anyway without `nlohmann_json` on Windows (or at least I think it was), so was waiting for results. Tthanks for checking and fixing!